hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -49,6 +49,9 @@
   @Parameter(names = {"--jdbc-url"}, description = "Hive jdbc connect url", required = true)
   public String jdbcUrl;
 
+  @Parameter(names = {"--metastore-uri"}, description = "Hive metastore uri,enable this configuration when synchronizing to remote hive)")
```

Comment:
White space between `uri,enable`, Or change to `Hive meta store URI which enables synchronize to xxx`